Polyfill/ReturnTypeWillChange.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+if (!class_exists('ReturnTypeWillChange', false)) {
+    #[\Attribute(\Attribute::TARGET_METHOD)]
+    final class ReturnTypeWillChange
+    {
+        public function __construct() {}
+    }
+}
```

Comment:
Shoot! I had composer.json changes for the autoloading, but must have reverted them accidentally.

Reply:
In php >=8.0, it will hit the attribute and then the autoloader will be triggered.
The polyfill is needed because ReturnTypeWillChange does not exist in Php 8.0

Reply:
https://3v4l.org/uUB9g

Oh, I see now. Attributes are never really loaded...